connectors/package-lock.json

```diff
@@ -86,6 +86,9 @@
       "version": "0.1.0",
       "dependencies": {
         "@notionhq/client": "^2.2.4",
+        "cheerio": "^1.0.0-rc.12",
+        "csv-parse": "^5.5.6",
```

Comment:
We don't need theses dependencies right? Given that the logic lives in core now.

Reply:
`csv-parse` is a `types` dependencies, right?